test/e2e/framework/pod/node_selection.go

```diff
@@ -48,6 +48,28 @@ func setNodeAffinityRequirement(nodeSelection *NodeSelection, operator v1.NodeSe
 		})
 }
 
+// SetNodeAffinityTopologyRequirement sets node affinity to a specified topology
+func SetNodeAffinityTopologyRequirement(nodeSelection *NodeSelection, topology map[string]string) {
```

Comment:
This function is always called with empty `nodeSelection`. Can it return newly created `*NodeSelection` instead?

Reply:
It's supposed to be the same as in the other functions in the file: they're also called with empty `nodeSelection`. This looks more universal.

Reply:
Ack